src/backend/InvenTree/users/test_tokens.py

```
@@ -0,0 +1,68 @@
+
+    def test_admin_action_disable_v1(self):
+        from users.admin import ApiTokenAdmin
+        from django.contrib.admin.sites import AdminSite
+        from django.test import RequestFactory
+        from django.contrib.messages.storage.base import BaseStorage
+        ApiToken.objects.create(user=self.user, key=ApiToken.generate_key(), version=1)
+        ApiToken.objects.create(user=self.user, key=ApiToken.generate_key(), version=2)
+        self.assertEqual(ApiToken.objects.filter(version=1, revoked=False).count(), 1)
+        admin = ApiTokenAdmin(ApiToken, AdminSite())
+        request = RequestFactory().get('/')
+        setattr(request, '_messages', BaseStorage(request))
```

Ruff (0.15.15)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)